rllib/agents/trainer.py

```diff
@@ -493,7 +496,12 @@ def _setup(self, config):
                 self.env_creator = _global_registry.get(ENV_CREATOR, env)
             else:
                 import gym  # soft dependency
-                self.env_creator = lambda env_config: gym.make(env)
+                env_ = gym.make(env)
+                act_wrap = NormalizeActionWrapper
+                if config["normalize_actions"]:
+                    env_ = act_wrap(env_.env)
+                self.env_creator = lambda env_config: env_
```

Comment:
Can you make it so this wrapper works for both the gym and non-gym cases?

Reply:
`self.env_creator` is a lambda function. When the env is not from gym, not sure on how to modify the environment before getting the env_config parameter.

Reply:
Why not

```
self.env_creator = ...  # anything

if config["normalize_actions"]:
   old_creator = self.env_creator

   def make_normalized(env_config):
       env = old_creator(env_config)
       assert isinstance(env, gym.Env)
       env = NormalizeActionWrapper(env)
       return env

   self.env_creator = make_normalized
```